### ISSUE TITLE
dyld: Library not loaded: @rpath/libswiftCore.dylib

### ISSUE THREAD

**Issue by smockle** [LinusU/node-dark-mode-listener#1]
(_**Background:** When I realized https://github.com/LinusU/vscode-auto-dark-mode wasn’t working for me (no change  in VS Code when the macOS appearance changes), I started debugging._)

I created a test project using this library (`node-dark-mode-listener`). My test project also did not log changes when the macOS appearance changed (e.g. dark → light).

I reviewed this library’s source and realized it basically wraps a binary. I tried executing the binary directly, and got the following error:

```
$ ./dark-mode-listener 
dyld: Library not loaded: @rpath/libswiftCore.dylib
  Referenced from: /Users/clay/Developer/foo/node_modules/dark-mode-listener/./dark-mode-listener
  Reason: image not found
Abort trap: 6
```

I found a lot of documentation about that error (e.g. [this StackOverflow question](https://stackoverflow.com/questions/26024100/dyld-library-not-loaded-rpath-libswiftcore-dylib)), but I haven’t attempted to resolve it yet.

(Actually, I didn’t realize the source for the `dark-mode-listener` binary is available until I started writing this issue, but now that I’ve located it in https://github.com/LinusU/DarkModeListener, I will attempt to fix the issue I’m reporting here.)

**Comment by smockle** [comment]
Possibly related:
- https://github.com/yonaskolb/Mint/pull/70
- https://github.com/yonaskolb/Mint/issues/69

**Comment by LinusU** [comment]
Ah, we could probably just add `-Xswiftc -static-stdlib` when building the binary

edit: hmm, one drawback of that is that the binary grows from just 69.5 KB to almost 10MB 😱

**Comment by LinusU** [comment]
@smockle can you see if c1ebc80 works for you?

**Comment by smockle** [comment]
@LinusU Yup, that works:
```
$ npm i --save https://github.com/LinusU/node-dark-mode-listener
$ cd ./node_modules/dark-mode-listener/
$ ./dark-mode-listener 
light
```

Update: Tested on two Macs—one with Xcode, the other without. Works on both. :+1:

**Comment by LinusU** [comment]
Great, releasing a new version now!

**Comment by smockle** [comment]
Many thanks! 💯

**Comment by LinusU** [comment]
Released as 0.1.1, will release a new version of the extension asap

**Comment by LinusU** [comment]
Fixed in the 0.1.4 version of the extension 🚀
